_sass/_layout.scss

```diff
@@ -55,21 +51,20 @@
         text-decoration: none;
         padding: 5px 15px 5px 15px;
         margin-left: 1px;
-        color: white;
+        color: $scheme-color-1;
```

Comment:
I wonder whether this and the changes below are really necessary: We could just keep the current colors for the menu so that the site does not become "all white".